site/content/admin-guide/mqtt-adapter-config.md

```diff
@@ -23,6 +23,7 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_MQTT_CERT_PATH`<br>`--hono.mqtt.certPath` | no | - | The absolute path to the PEM file containing the certificate that the protocol adapter should use for authenticating to clients. This option must be used in conjunction with `HONO_MQTT_KEY_PATH`.<br>Alternatively, the `HONO_MQTT_KEY_STORE_PATH` option can be used to configure a key store containing both the key as well as the certificate. |
 | `HONO_MQTT_COMMAND_ACK_TIMEOUT`<br>`--hono.mqtt.commandAckTimeout` | no | `100` | The amount of time (milliseconds) after which the sending of a command to a device using QoS 1 is considered to be failed. The value of this variable should be increased in cases where devices are connected over a network with high latency. |
 | `HONO_MQTT_DEFAULTS_ENABLED`<br>`--hono.mqtt.defaultsEnabled` | no | `true` | If set to `true` the protocol adapter uses *default values* registered for a device to augment messages published by the device with missing information like a content type. In particular, the protocol adapter adds default values registered for the device as (application) properties with the same name to the AMQP 1.0 messages it sends downstream to the Hono Messaging service. |
+| `HONO_MQTT_MAX_CONNECTIONS`<br>`--hono.mqtt.maxConnections` | no | *matching memory* | The number of maximal concurrent connections the protocol adapter should accept. If not set (or set to `0`), the protocol adapter calculates at startup how many connections it can serve with the given amount of memory. |
```

Comment:
The idea here was to list the configuration variables in alphabetical order. Can you move it right before `HONO_MQTT_MAX_PAYLOAD_SIZE`?

Reply:
Sorry, should have seen it.